src/components/data_search/DatasetSearchTable.js

```diff
@@ -314,6 +312,9 @@ export const DatasetSearchTable = (props) => {
                         </Link>);
                     }
                   },
+                  {
+                    value: dataset.accessManagement==='external' ? 'External to DUOS': dataset.accessManagement==='open'? 'Open Access':dataset.dac?.dacEmail ? <Link href={'mailto:' + dataset.dac.dacEmail}>{dataset.dac?.dacName}</Link> : dataset.dac?.dacName,
+                  },
```

Comment:
I think it might be clearer to have this as a function with `if else` statements. Also, we should switch the column names, not the values, since we want the exportable snapshot link to be next to the location.

Reply:
@fboulnois I reverted the switch of the values because I don't fully understand the bug with that column yet. I'll fix that in a different PR.